packages/storage/src/metadata/gundbMetadataStore.ts

```diff
@@ -478,6 +499,10 @@ export class GundbMetadataStore implements UserMetadataStore {
     });
   }
 
+  private getNotificationsLastSeenAtLookupKey(): string {
+    return `notifications/lastSeenAt/${this.username}`;
```

Comment:
```suggestion
    return `notifications/lastSeenAt`;
```

Since the key already namespaced to the user, we shouldn't need to add the username field to the lookup key